modules/graph/loader/basic_ev_fragment_loader.h

```diff
@@ -88,7 +93,7 @@ class BasicEVFragmentLoader {
   /// Set attributes: vertex_label_num_, vm_ptr_, output_vertex_tables_
   ///                 vertex_label_to_index_, vertex_labels_
   boost::leaf::result<void> ConstructVertices(
-      ObjectID vm_id = InvalidObjectID()) {
+      ObjectID vm_id = InvalidObjectID(), bool with_local_vertex_map = false) {
```

Comment:
We already have a type paramter `VERTEX_MAP_T`, why the runtime paramter `with_local_vertex_map` is still needed?

Reply:
the `LocalVertexMap` construct logic is not same with `GlobalVertexMap`, and it's not easy to unify them.